pom.xml

```diff
@@ -248,6 +248,35 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.ec4j.maven</groupId>
+					<artifactId>editorconfig-maven-plugin</artifactId>
+					<version>0.1.3						                        						                        						<!-- check the latest on http://search.maven.org/maven2/org/ec4j/maven/editorconfig-maven-plugin/ --></version>
+					<executions>
```

Comment:
Remove these comments - they are just there to begin with.

Reply:
Deleted comment